hal/armv7m/imxrt/interrupts.c

```diff
@@ -18,14 +18,17 @@
 #include "cpu.h"
 #include "pmap.h"
 
-#ifdef CPU_IMXRT1170
-#include "imxrt1170.h"
+#ifdef CPU_IMXRT117X
 #define SIZE_INTERRUPTS 217
-#else
-#include "imxrt.h"
+#include "imxrt117x.h"
+#endif
+
+#if defined(CPU_IMXRT105X) || defined(CPU_IMXRT106X)
 #define SIZE_INTERRUPTS 167
```

Comment:
Maybe we should move SIZE_INTERRUPTS to the platform header

Reply:
I agree with you. The right place for this is the platform header.